docs/advanced-topics/client-anticipation.md

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
Little nitpick about the images. Not a blocker, but could make them a bit clearer. I'd try to put on top the "main" lesson" and have the other "support" client at the bottom. So for example, if the lesson is about reanticipation, I'd put the client timeline doing the reanticipation on top. Makes it easy to jump at the right spot when reading.
Not crazy important, as a reader I can figure it out, it just requires a bit more concentration.